### PR TITLE
this fixes and closes issue #46

### PR DESCRIPTION
**Pull request by Jummate**
### What does this PR do?:

Fixed bugs due to improper alignment and duplicates in contributors' list

### Description of Task to be completed?

- Re-arranged the table into rows of 7 columns

- Removed the duplicates in contributors' list

### CODE REVIEW

**Review by Evavic44 — COMMENTED** [review]
Hello, @Jummate can you kindly only remove the duplicate and leave other profiles as is. This makes reviewing easier. Thanks 👍🏾

**Comment by Jummate** [comment]
@Evavic44 
All right boss. I am on it.

File: Contributors.md [hunk 1]
```diff
@@ -157,18 +157,7 @@
   </a>
 </td>
 <!-- End of column-13 -->
-<!-- Start of column-14 -->
```
Comment (Evavic44) [inline Contributors.md:160]: Is this profile on `<!-- Start of column-14 -->` (Olasunkanmi Balogun) a duplicate?

**Comment by Evavic44** [comment]
Nice work @Jummate will fix the rest. 🙌🏾

**Pull request merged**